build/teamcity/cockroach/nightlies/stress_impl.sh

```diff
@@ -15,40 +15,29 @@ bazel build //pkg/cmd/bazci --config=ci
 BAZEL_BIN=$(bazel info bazel-bin --config=ci)
 ARTIFACTS_DIR=/artifacts
 
-# Query to list all affected tests.
-PKG=${PKG#"./"}
-if [[ $(basename $PKG) != ... ]]
+if [[ ! -z $(bazel query "attr(tags, \"broken_in_bazel\", $TARGET)") ]]
 then
-    PKG="$PKG:all"
+    echo "Skipping test $TARGET as it is broken in bazel"
+    exit 0
+fi
+if [[ ! -z $(bazel query "attr(tags, \"integration\", $TARGET)") ]]
+then
+    echo "Skipping test $TARGET as it is an integration test"
+    exit 0
+fi
+exit_status=0
+GOTESTTIMEOUTSECS=$(($TESTTIMEOUTSECS - 5))
+$BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci -- test --config=ci "$TARGET" \
+                                        --test_env=COCKROACH_NIGHTLY_STRESS=true \
+                                        --test_timeout="$TESTTIMEOUTSECS" \
+                                        --test_arg=-test.timeout="${GOTESTTIMEOUTSECS}s"
+                                        --run_under "@com_github_cockroachdb_stress//:stress -bazel -shardable-artifacts 'XML_OUTPUT_FILE=$BAZEL_BIN/pkg/cmd/bazci/bazci_/bazci merge-test-xmls' $STRESSFLAGS" \
+                                        --define "gotags=$TAGS" \
+                                        --nocache_test_results \
+                                        --test_output streamed \
+                                        ${EXTRA_BAZEL_FLAGS} \
+    || exit_status=$?
```

Comment:
Nit: I think this and the `if [ $exit_status -ne 0 ]` bit can be omitted now, because if the `bazci` command failed, the shell script will just fail with the same error.